diff --git a/OpenBoard.pro b/OpenBoard.pro
index 8798e432..a0a4abf7 100644
--- a/OpenBoard.pro
+++ b/OpenBoard.pro
@@ -71,6 +71,9 @@ include(src/pdf-merger/pdfMerger.pri)
 include(plugins/plugins.pri)
 INCLUDEPATH += plugins/cffadaptor/src
 
+#ThirdParty
+INCLUDEPATH += /usr/include/quazip5/
+
 FORMS += resources/forms/mainWindow.ui \
    resources/forms/preferences.ui \
    resources/forms/brushProperties.ui \
@@ -434,7 +437,7 @@ linux-g++* {
     #LIBS += -lprofiler
     LIBS += -lX11
     LIBS += -lquazip5
-    INCLUDEPATH += "/usr/include/quazip"
+    INCLUDEPATH += "/usr/include/quazip5"
 
     LIBS += -lpoppler
     INCLUDEPATH += "/usr/include/poppler"
diff --git a/plugins/cffadaptor/UBCFFAdaptor.pro b/plugins/cffadaptor/UBCFFAdaptor.pro
index f8214d5d..315d72e3 100644
--- a/plugins/cffadaptor/UBCFFAdaptor.pro
+++ b/plugins/cffadaptor/UBCFFAdaptor.pro
@@ -39,7 +39,7 @@ macx:{
     QMAKE_POST_LINK += bash postScript_mac.sh "$$DESTDIR" "$$SANKORE_DIR/$$BUILD_DIR/product/Open-Sankore.app/Contents/MacOS"
 }
 
-LIBS     += "-L$$THIRD_PARTY_PATH/quazip/lib/$$SUB_DIR" "-lquazip"
+LIBS     += "-lquazip5"
 
 QT       += xml xmlpatterns core
 QT       += gui
@@ -51,9 +51,7 @@ DEFINES += NO_THIRD_PARTY_WARNINGS
 
 INCLUDEPATH += src
 
-DEPENDPATH  += $$THIRD_PARTY_PATH/quazip/
-INCLUDEPATH += $$THIRD_PARTY_PATH/quazip/
-include($$THIRD_PARTY_PATH/quazip/quazip.pri)
+INCLUDEPATH += /usr/include/quazip5/
 
 
 SOURCES += \
